app/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = yeoman.generators.Base.extend({
     this.directory('.', '.');
   },
 
+  generateYoRc: function() {
+    this.log('Generating .yo-rc.json');
+    this.config.save();
+  },
+
```

Comment:
.yo-rc.json allows the boot script command (any yo command (AFAIK) from any subdir in the project. This means we can run `yo loopback:boot-script` without actually being in the project root (because yeoman looks for `.yo-rc.json` to determine the project root). IMO this should've been there a long time ago. Opinions?